common/dynamicconfig/constants.go

```diff
@@ -115,6 +115,8 @@ const (
 	DeadlockInterval = "system.deadlock.Interval"
 	// How many extra goroutines can be created per root.
 	DeadlockMaxWorkersPerRoot = "system.deadlock.MaxWorkersPerRoot"
+	// Enable / disable worker versioning.
+	EnableWorkerVersioning = "system.workerVersioning.enable"
```

Comment:
this should be something like `frontend.workerVersioningData` since it's controlling the frontend rpcs that read/write versioning data. we'll have another dynamic config for the matching behavior